activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/service/ProjectService.java

```diff
@@ -215,21 +214,35 @@ public Project importProject(MultipartFile file) throws IOException {
     }
 
     public void validateProject(Project project) {
+        List<ModelValidationError> validationErrors = new ArrayList<>();;
         List<Model> availableModels = modelService.getAllModels(project);
-        ValidationContext validationContext = new ProjectValidationContext(availableModels);
 
-        List<ModelValidationError> validationErrors = availableModels
-                .stream()
-                .flatMap(model -> getModelValidationErrors(model,
-                                                           validationContext))
+        if (isProjectEmpty(availableModels)) {
+            validationErrors.add(getEmptyProjectError());
```

Comment:
The code is ok, but you can create directly the list with one element. The following looks cleaner, IMO:
```
List<ModelValidationError> validationErrors = isProjectEmpty(availableModels) ?
    Collections.singletonList(getEmptyProjectError()) : 
    availableModels
                .stream()
                .flatMap(model -> getModelValidationErrors(model, 
                                                           new ProjectValidationContext(availableModels)))
                .collect(Collectors.toList());
```

Reply:
Thank you, change request fixed, please review if it is ok now.